modules/topicsFpdModule.js

```diff
@@ -19,6 +19,9 @@ const bidderIframeList = {
   bidders: [{
     bidder: 'pubmatic',
     iframeURL: 'https://ads.pubmatic.com/AdServer/js/topics/topics_frame.html'
+  }, {
+    bidder: 'rtbhouse',
+    iframeURL: 'https://topics.creativecdn.com/tapi.html'
```

Comment:
this frame doesnt post to top.

Reply:
take a look at view-source:https://ads.pubmatic.com/AdServer/js/topics/topics_frame.html

Reply:
Appreciate the early feedback @patmmccann ! 
Apologies for too-early post - did get roll back to WIP, but you were faster :-) 
Will update in the next few days

Reply:
If you study the latest pr from @jlquaccia , it might make sense to branch off that and avoid the iframe altogether

Reply:
Thanks for the pointer! 
I could be wrong here, but iframe could make sense as the call would be appropriated to the iframe's domain, rather than to the publisher's domain like in 1-party-context javascript call on pub domain. 
If that's fine with you - I'd prefer sticking with iframe for now

Reply:
Apologies, marking as a draft again